packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -70,11 +74,13 @@ contract L1Block is ISemver {
         uint64 _sequenceNumber,
         bytes32 _batcherHash,
         uint256 _l1FeeOverhead,
-        uint256 _l1FeeScalar
+        uint256 _l1FeeScalar,
+        uint256[] calldata _dependencySet
```

Comment:
No need to modify this function - this function needs to not change for pre-hardfork reasons

Reply:
ahh ok got it